src/emulator/host/src/config.cpp

```diff
@@ -126,6 +127,7 @@ ExitCode deserialize(Config &cfg) {
     get_yaml_value(config_node, "log-active-shaders", &cfg.log_active_shaders, false);
     get_yaml_value(config_node, "log-uniforms", &cfg.log_uniforms, false);
     get_yaml_value(config_node, "pstv-mode", &cfg.pstv_mode, false);
+    get_yaml_value(config_node, "show-gui", &cfg.show_gui, false);
```

Comment:
```suggestion
    get_yaml_value(config_node, "show-gui", &cfg.show_gui, true);
```

Reply:
ah for that no, i want leave it on false by default, for all user is better hide gui in defaut setting

Reply:
Aight I guess it's fine if we can still enable it from the settings.